deliver/lib/deliver/submit_for_review.rb

```diff
@@ -63,6 +63,17 @@ def update_export_compliance(options, app, build)
 
       UI.verbose("Updating build for export compliance status of '#{uses_encryption}'")
       if build.uses_non_exempt_encryption.nil?
+        if uses_encryption.to_s.empty?
```

Comment:
@joshdholtz in the issue #16645 the `export_compliance_uses_encryption` was set in accordance with http://docs.fastlane.tools/actions/deliver/#compliance-and-idfa-settings, but still got the error. It's as if it never got into this if.

Reply:
I added more details to the issue https://github.com/fastlane/fastlane/issues/16645#issuecomment-648032084

Reply:
This code is still not in prod, unless you have checked out the branch, but let's focus on reviewing the PR, we can discuss about issues in their places 👍

Reply:
@minuscorp Sure, this PR looks good on its own, but it looks that it doesn't address the issue which motivated it. And yes, I'm testing with the 2.150.0.rc1 version :)

Reply:
Did you find where the `submission_information[:export_compliance_uses_encryption]` is being set?

Reply:
Yes, that comes from the deliver config https://github.com/artsy/eigen/blob/faa02e2746194d8d7c11899474de9c517435eca4/fastlane/Fastfile#L131-L149

Reply:
I meant, internally

Reply:
Oh, I get it, no I didn't find that...